test/test_mps.py

```diff
@@ -4013,27 +4029,28 @@ def helper(n, c, h, w):
     def test_divmode(self):
         def helper(shape, rounding_mode):
             for dtype in [torch.float32, torch.float16, torch.int32, torch.int64]:
-                cpu_x = None
-                cpu_y = None
-                if (dtype in [torch.float32, torch.float16]):
-                    cpu_x = torch.randn(shape, device='cpu', dtype=dtype, requires_grad=False)
-                    cpu_y = torch.randn(shape, device='cpu', dtype=dtype, requires_grad=False)
-                else:
-                    cpu_x = torch.randint(-10, 0, shape, device='cpu', dtype=dtype, requires_grad=False)
-                    cpu_y = torch.randint(-10, 0, shape, device='cpu', dtype=dtype, requires_grad=False)
+                if (rounding_mode != None and "floor" in rounding_mode and dtype == torch.int64) is None:
```

Comment:
I think you'll have to do something like this to pass for lint:
```suggestion
                if (rounding_mode is not None and "floor" in rounding_mode and dtype == torch.int64) is None:
```